axelor-account/src/main/resources/views/PaymentSession.xml

```diff
@@ -131,8 +131,8 @@
     </panel>
     <panel name="emailPanel" sidebar="true" showIf="id &amp;&amp; sessionTotalAmount > 0">
       <button name="sendEmailsBtn" title="Send emails payment notice"
-        showIf="statusSelect == 1" readonlyIf="hasEmailsSent == true"
-        onClick="save,action-payment-session-method-send-emails"/>
+        showIf="statusSelect == 1 || statusSelect == 2 || statusSelect == 3"
```

Comment:
Adil asked for: 
"Allow the sending of payment notices by email only from the status " awaiting payment " and " closed ""

Here there is still "statusSelect == 1" in the showIf, and 1 is Ongoing, it must be removed